docs/apis/http/graph/spaces.md

```diff
@@ -638,3 +638,130 @@ The space to be deleted was not disabled before.
 ```
 {{< /tab >}}
 {{< /tabs >}}
+
+## Sharing Space
+
+### Add Member to Space `POST /drives/{drive-id}/root/invite`
+
+Share a space 
+
+{{< tabs "share-drive" >}}
+{{< tab "Request" >}}
+```shell
+curl -L -k -X POST 'https://localhost:9200/graph/v1beta1/drives/b14b87f3-dbc7-43f4-83fb-c9d6674ac776%24f29eeb57-2b8d-448b-bacc-59f40168ec59/root/invite' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "roles": [
+        "a8d5fe5e-96e3-418d-825b-534dbdf22b99"
+    ],
+    "recipients": [
+        {
+            "objectId": "f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c",
+            "@libre.graph.recipient.type": "user"
+        }
+    ]
+    "expirationDateTime":"2024-07-31T23:59:59.999+05:45",
+}'
+```
+{{< hint type=info title="Body value" >}}
+
+This recipients can be a user or a group. You should provide the following body when sharing with a group
+
+```
+...
+"recipients": [
+        {
+            "objectId": "{group-id}",
+            "@libre.graph.recipient.type": "group"
+        }
+    ]
+...
+```
+
+{{< /hint >}}
+{{< /tab >}}
+{{< tab "Response - 200 OK" >}}
+```json
+{
+    "value": [
+        {
+            "expirationDateTime":"2024-07-31T18:14:59.999Z",
+            "grantedToV2": {
+                "user": {
+                    "displayName": "Marie Skłodowska Curie",
+                    "id": "f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c"
+                }
+            },
+            "id": "u:f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c",
+            "invitation": {
+                "invitedBy": {
+                    "user": {
+                        "displayName": "Admin",
+                        "id": "adef50cd-d37f-4532-87cd-7f98b2584751"
+                    }
+                }
+            },
+            "roles": [
+                "a8d5fe5e-96e3-418d-825b-534dbdf22b99"
+            ]
+        }
+    ]
+}
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+### Add Member to Space (permissions) `POST /drives/{drive-id}/items/{item-id}/invite`
+
+If you are adding a member to space you should use the previous API or you have to make an extra `PROPFIND` request to get the item-id.
```

Comment:
I would say that we use this endpoint to share space items. but we also can share space using this endpoint and itemId=spaceId

> ### Add Member to Space (permissions)

what means `permissions` here? maybe you wanted to say that we can share space using rolesId or array of the permissions?

Reply:
itemid: space root file-id